ipn/serve.go

```diff
@@ -37,12 +37,11 @@ type ServeConfig struct {
 	// traffic is allowed, from trusted ingress peers.
 	AllowFunnel map[HostPort]bool `json:",omitempty"`
 
-	// Foreground is a map of an IPN Bus session id to a
-	// foreground serve config. Note that only TCP and Web
-	// are used inside the Foreground map.
-	//
-	// TODO(marwan-at-work): this is not currently
-	// used. Remove the TODO in the follow up PR.
+	// Foreground is a map of an IPN Bus session ID to a
+	// foreground serve config. This allows the config to specify
```

Comment:
... to an alternate foreground serve config that's valid for the life of that WatchIPNBus session ID. This allows ...